queries/typescript/textobjects.scm

```diff
@@ -1 +1,32 @@
 ; inherits: ecma
+
+
+;; interface declaration as class textobject
+(interface_declaration) @class.outer
+
+(interface_declaration
+  body: _ @class.inner) @class.outer
+
+(interface_declaration
+  body: (object_type . "{" . _ @_start @_end _? @_end . "}"
```

Comment:
it works for me when you remove the first `@_end`

Reply:
Has it been resolved?